python_modules/libraries/dagster-docker/dagster_docker/utils.py

```diff
@@ -60,15 +59,3 @@ def validate_docker_config(network, networks, container_kwargs):
             raise Exception(
                 "'network' cannot be used in 'container_kwargs'. Use the 'network' config key instead."
             )
-
-
-def validate_docker_image(docker_image):
-    try:
-        # validate that the docker image name is valid
-        reference.Reference.parse(docker_image)
```

Comment:
Do we want to remove this entirely or replace it with a rewrite using the standard library? Will it be obvious why the run/step fails if you pass an invalid image name or will we get some very cryptic error instead?